moveit_simple/src/moveit_simple.cpp

```diff
@@ -599,8 +677,10 @@ bool Robot::getFK(const std::vector<double> & joint_point,
                   Eigen::Affine3d &pose) const
 {
   robot_state_->setJointGroupPositions(joint_group_, joint_point);
-  const std::vector<std::string> link_names = robot_model_ptr_->getLinkModelNames();
-  if (!link_names.empty())
+  const std::vector<std::string> link_names = joint_group_->getLinkModelNames();
+  const std::vector<std::string> active_joints = joint_group_->getActiveJointModelNames();
+  const int vc =  (int)robot_state_->getVariableCount();
+  if ( active_joints.size() == vc)
```

Comment:
This should always be true.  However, you don't check the `joint_point` parameter, which could be inconsistent.